src/bloom/BloomFilter.ts

```diff
@@ -34,16 +53,45 @@ export class BloomFilter extends AlphaFilter
     public static readonly DEFAULT_OPTIONS: BloomFilterOptions = {
         strength: { x: 2, y: 2 },
         quality: 4,
+        resolution: 1,
         kernelSize: 5
     };
 
     private _blurXFilter: BlurFilterPass;
     private _blurYFilter: BlurFilterPass;
     private _strength: PointData;
 
-    constructor(options?: BloomFilterOptions)
+    constructor(options?: BloomFilterOptions);
+    /**
+    * @deprecated since 6.0.0
+    *
+    * @param {number|PIXI.PointData|number[]} [blur=2] - Sets the strength of both the blurX and blurY properties simultaneously
+    * @param {number} [quality=4] - The quality of the blurX & blurY filter.
+    * @param {number} [resolution=1] - The resolution of the blurX & blurY filter.
+    * @param {number} [kernelSize=5] - The kernelSize of the blurX & blurY filter.Options: 5, 7, 9, 11, 13, 15.
+    */
+    constructor(blur?: DeprecatedBlurValue, quality?: number, resolution?: number, kernelSize?: number);
+    constructor(...args: [BloomFilterOptions?] | [DeprecatedBlurValue?, number?, number?, number?])
     {
-        options = { ...BloomFilter.DEFAULT_OPTIONS, ...options };
+        let options = args[0] ?? {};
+
+        if (typeof options === 'number' || Array.isArray(options) || ('x' in options && 'y' in options))
+        {
+            // eslint-disable-next-line max-len
+            deprecation(v8_0_0, 'BloomFilter constructor params are now options object. See params: { strength, quality, resolution, kernelSize }');
```

Comment:
Can you please change all the deprecation notices and docs to be 6.0.0? There are still a bunch of them here.
```suggestion
            deprecation('6.0.0', 'BloomFilter constructor params are now options object. See params: { strength, quality, resolution, kernelSize }');
```

Reply:
sure thing!